extensions/pvaPublish/src/node/publish.ts

```diff
@@ -92,23 +92,37 @@ export const publish = async (
         'If-Match': project.eTag,
       },
     });
-    const job: PVAPublishJob = await res.json();
-    logger.log('Publish job started: %O', job);
+    if (res.status === 202) {
+      const job: PVAPublishJob = await res.json();
+      logger.log('Publish job started: %O', job);
 
-    // transform the PVA job to a publish response
-    const result = xformJobToResult(job);
+      // transform the PVA job to a publish response
+      const result = xformJobToResult(job);
 
-    // add to publish history
-    const botProjectId = project.id;
-    ensurePublishProfileHistory(botProjectId, profileName);
-    publishHistory[botProjectId][profileName].unshift(result);
+      // add to publish history
+      const botProjectId = project.id;
+      ensurePublishProfileHistory(botProjectId, profileName);
+      publishHistory[botProjectId][profileName].unshift(result);
 
-    logger.log('Publish call successful.');
+      logger.log('Publish call successful.');
 
-    return {
-      status: result.status,
-      result,
-    };
+      return {
+        status: result.status,
+        result,
+      };
+    } else {
```

Comment:
The if clause only checks for 202.  Would this else clause report an error for a 200 if there was text in the result?  Should this be an else if clause looking for error codes?

Reply:
Correct. We don't expect any other status codes back, so if we get a 200 back then something went wrong.

Here is the [actual route on PVA's side for this endpoint](https://msazure.visualstudio.com/CCI/_git/BotDesigner?path=%2Fsrc%2FBotDesigner.Management.Server%2FBotDesigner.Citizen.Server%2FBotDesigner.Management.Api%2FControllers%2FComposerContentManagementController.cs&version=GC708b173f4bab91833fa7e97cd1fca7876cb4e4c4&line=63&lineEnd=64&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents). You can see that it is registered to return a 202 Accepted.